src/herder/test/TestTxSetUtils.h

```diff
@@ -0,0 +1,24 @@
+// Copyright 2022 Stellar Development Foundation and contributors. Licensed
+// under the Apache License, Version 2.0. See the COPYING file at the root
+// of this distribution or at http://www.apache.org/licenses/LICENSE-2.0
+
+#pragma once
+
+#include "herder/TxSetUtils.h"
+
+namespace stellar
+{
+namespace TxSetUtils
```

Comment:
do not use the prod namespace, just call it `TxSetTestUtils`

Reply:
Fixed. See above.